design-documents/graph-ql/coverage/Wishlist.graphqls

```diff
@@ -7,8 +7,9 @@ type Mutation {
 }
 
 type Customer {
-    wishlist: Wishlist! @doc(description: "Customer wishlist") # Commerce will extend filed with required `id` argument `wishlists(ids: ID!)`
-    wishlists: [Wishlist!]! @doc(description: "Customer multiple wishlists") # Multiple wishlists Commerce functionality
+    wishlist: Wishlist! @deprecated(reason: "Use `Customer.wishlists` or `Customer.wishlistByID") 
+    wishlistByID(id: ID!): Wishlist
+    wishlists: [Wishlist!]! @doc(description: "Customer wishlists are limited to a max of 1 wishlist in Magento Open Source")
```

Comment:
Can also be limited to one in Commerce, with the `Enable Multiple Wish Lists` feature set to false

https://docs.magento.com/user-guide/configuration/customers/wishlist.html#general-options